epitome/list_genomes.py

```diff
@@ -0,0 +1,6 @@
+import boto3
+
+s3client = boto3.client('s3', region_name='us-west-1')
```

Comment:
do we need this?

Reply:
Didn't realize this file got added, removed.